include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -1137,6 +1137,13 @@ FUNCTION(GetErrorValue, swift_getErrorValue, C_CC,
          ARGS(ErrorPtrTy, Int8PtrPtrTy, OpenedErrorTriplePtrTy),
          ATTRS(NoUnwind))
 
+// void __tsan_write8(void *addr);
+// This is a Thread Sanitizer instrumentation entry point in compiler-rt.
+FUNCTION(TSanInoutAccess, __tsan_write8, C_CC,
+         RETURNS(VoidTy),
+         ARGS(Int8PtrTy),
+         ATTRS(NoUnwind))
```

Comment:
Is the plan to eventually use an appropriately sized `__tsan_writeN` call? Is there a dynamically-sized one for generic code?